haproxy-ingress/templates/_podtemplate.yaml

```diff
@@ -115,7 +128,8 @@ spec:
         - --{{ $key }}
     {{- end }}
   {{- end }}
-      {{- include "haproxy-ingress.controller.ports" . | nindent 6 }}
+      ports:
+        {{- include "haproxy-ingress.controller.ports" . | nindent 8 }}
```

Comment:
Change here as well if choosing to unindent the include.

Reply:
Unindenting the ports template fixes this.